waving_hands/gamemaster.py

```diff
@@ -4056,4 +4082,5 @@ def main():
     gm.setup_game()     # create wizards, customize, etc
     gm.play_game()      # see play_game() function for further details
 
-main()
\ No newline at end of file
+if __name__ == '__main__':
+    main()
```

Comment:
`if __name__ == '__main__'` is another handy little trick if you find yourself writing modules you want to call explicitly like `python gamemaster.py`. Any code that imports this module will skip this block, allowing the `waving_hands.py` client to import it without immediately starting the server.